### PR TITLE
Hide overview tab on /careers/all and add 404 & 500 pages

### LINKED ISSUE #108: Careers - Remove the overall tab from the all roles

### PR DESCRIPTION
**Pull request by solazio**
## Done

- Hide `overview` tab on `/careers/all`
- Add 404 & 500 pages
- Remove secondary nav on all `/careers` pages
- Add drop down for mobile navigation
- Update image on `/careers/travel`
- Make navigation sticky on home page
- Remove navigation drop downs on home page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/careers/all
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Make sure the overview tab is not visible
- Go to http://0.0.0.0:8002/404 and http://0.0.0.0:8002/500 and see a sensible content on the page

## Issue / Card

Fixes #108

### CODE REVIEW

**Comment by waynecrosby** [comment]
@solazio On mobile the dropdown arrows have disappeared from the Nav menu

File: templates/partial/_navigation.html [hunk 1]
```diff
@@ -57,10 +68,12 @@
...
   var nav = document.querySelector('.js-show-nav');
   var hash = window.location.hash;
-  nav.classList.remove('u-hide');
+  if (nav) {
+    nav.classList.remove('u-hide');
+  }
 
   // If the page loads with a preselected hash load and open the menu
-  if (hash) {
+  if ((hash) && (!((hash==="#products") || (hash="#careers")))) {
```
Comment (anthonydillon) [inline templates/partial/_navigation.html]: Your missing a few equals in the last conditionally in this statement

**Comment by waynecrosby** [comment]
@solazio Can you make the illustration a bit smaller, say 80% and add a little bit of white space below, so some margin above the footer.
Screenshot 2019-11-20 at 09 34 42

**Comment by waynecrosby** [comment]
@solazio Can you make the Canonical a little bigger too, so it has a min-height: 2rem; and also make the menu sticky on all pages. Also with the menu on the home page, can you make the background selected when you click or get to that area on the page. So when you click Products and it jumps down the background is highlighted but when you scroll off that area it goes away? Thanks

**Comment by solazio** [comment]
> Can you make the Canonical a little bigger too, so it has a min-height: 2rem; and also make the menu sticky on all pages. Also with the menu on the home page, can you make the background selected when you click or get to that area on the page. So when you click Products and it jumps down the background is highlighted but when you scroll off that area it goes away? Thanks

@waynecrosby I created a new issue for the above https://github.com/canonical-web-and-design/canonical.com/issues/112

**Pull request merged**